coderd/rbac/builtin_test.go

```diff
@@ -251,8 +251,8 @@ func TestRolePermissions(t *testing.T) {
 			Actions:  []rbac.Action{rbac.ActionCreate, rbac.ActionUpdate, rbac.ActionDelete},
 			Resource: rbac.ResourceRoleAssignment,
 			AuthorizeMap: map[bool][]authSubject{
-				true:  {admin},
-				false: {orgAdmin, orgMemberMe, otherOrgAdmin, otherOrgMember, memberMe, templateAdmin, userAdmin},
+				true:  {admin, userAdmin},
```

Comment:
I thought we got rid of `admin`?

Reply:
`admin` is just the name of the `owner` user. Forgot to rename the var :+1: